test/fuzz/gen.ml

```diff
@@ -36,14 +37,18 @@ let expr_available_1_i32 if_else expr ~locals ~stack env =
     ]
   in
   [ pair B.iunop_32 (const [ S.Nothing ])
+  ; if_else expr ~locals ~stack env
   ; pair B.itestop_32 (const [ S.Nothing ])
-  ; pair B.i64_extend_i32 (const [ S.Pop; S.Push (Num_type I64) ])
-  ; pair B.extend_32_i32 (const [ S.Nothing ])
   ; pair B.f32_convert_i32 (const [ S.Pop; S.Push (Num_type F32) ])
   ; pair B.f64_convert_i32 (const [ S.Pop; S.Push (Num_type F64) ])
   ; pair B.f32_reinterpret_i32 (const [ S.Pop; S.Push (Num_type F32) ])
-  ; if_else expr ~locals ~stack env
   ]
+  @ ( if env.Env.conf = Concrete then
+        [ pair B.extend_32_i32 (const [ S.Nothing ])
+        ; pair B.i64_extend_i32 (const [ S.Pop; S.Push (Num_type I64) ])
+        ; pair B.iunop_popcnt_32 (const [ S.Nothing ])
+        ]
+      else [] )
```

Comment:
Could you put all of them as before and let the fuzzer fail if they're not implemented ? We'll implemented them later.